README.md

```diff
@@ -41,31 +41,33 @@ Run test execution automatically and correct tests if any changes in layout happ
 [Features](https://github.com/jdi-testing/jdi-light/labels/feature)
 
 ## Useful Links
-Test site: <a href = "https://jdi-testing.github.io/jdi-light/" target = "a_blank"> https://jdi-testing.github.io/jdi-light/ </a>
+Test site: https://jdi-testing.github.io/jdi-light/
 
 Login to get access to all pages (Login: Roman; Password: Jdi1234)
 
-Html Elements examples: <a href = "https://jdi-testing.github.io/jdi-light/html5.html" target = "a_blank"> https://jdi-testing.github.io/jdi-light/html5.html </a>
+Html Elements examples: https://jdi-testing.github.io/jdi-light/html5.html
 
-Example Html elements tests: <a href = "https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-html-tests" target = "a_blank"> https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-html-tests </a>
+Example Html elements tests: https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-html-tests
 
-Example Scenario tests: <a href = "https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-examples" target = "a_blank"> https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-examples </a>
+Example Scenario tests: https://github.com/jdi-testing/jdi-light/tree/master/jdi-light-examples
 
-Site: <a href = "http://jdi.epam.com/" target = "a_blank"> http://jdi.epam.com/ </a>
+Site: http://jdi.epam.com/
 
-VK: <a href = "https://vk.com/jdi_framework" target = "a_blank"> https://vk.com/jdi_framework </a>
+VK: https://vk.com/jdi_framework
 
-Facebook: <a href = "https://www.facebook.com/groups/jdi.framework/" target = "a_blank"> https://www.facebook.com/groups/jdi.framework/ </a>
+Facebook: https://www.facebook.com/groups/jdi.framework/
 
 You can ask your questions on StackOverflow with [![htmlelements](https://img.shields.io/badge/stackoverflow-jdiframework-orange.svg?style=flat)](http://stackoverflow.com/questions/tagged/jdiframework) tag
 
-Get Help in active users Skype chat: <a href = "https://join.skype.com/clvyVvnZvWqc" target = "a_blank"> https://join.skype.com/clvyVvnZvWqc </a>
+Get Help in active users Skype chat: https://join.skype.com/clvyVvnZvWqc
```

Comment:
It doesn't connect to issue 990. And it causes a lot of new Codacy issues:
https://app.codacy.com/manual/jdi-testing/jdi-light/pullRequest?prid=4361169